server/static/js/profile_page.js

```diff
@@ -1,9 +1,11 @@
 require(
 ['ext/jquery', 'ext/underscore', 'ext/underscore.string', 'ext/bootstrap',
 'term', 'course', 'friend', 'util', 'user', 'user_course', 'prof', 'exam',
-'raffle_unlock', 'schedule', 'sign_in', 'work_queue', ],
+'raffle_unlock', 'schedule', 'sign_in', 'work_queue', 'scholarship',
+'ext/react'],
```

Comment:
I totally goofed on this, but I just remembered that a consequence of writing some code in React is we're going to be increasing the amount of JavaScript for the user to download. (We do do this at Khan Academy though.) How many additional gzipped bytes are we going to be sending down to the user? (Check the network tab of Chrome's dev tools.)